docs/filter-fasta.md

```diff
@@ -77,7 +77,7 @@ moPepGen fitlerFasta \
 This example here removes any peptide sequences if they appear in the given denylist.
 
 !!! warning:
-When using noncoding peptides as denylist, do not also pass it as a input FASTA, because all peptides will be removed.
+When using noncoding peptides in a denylist, do not also pass it as an input FASTA, because all peptides will be removed.
```

Comment:
```suggestion
When using noncoding peptides in a denylist, do not also pass the noncoding peptide FASTA as an input FASTA, because all peptides will be removed.
```